test/k8s-integration/prepull-image.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+if [[ -z "${PREPULL_IMAGE}" ]]; then
+    # Pre-pull all the test images. The images are currently hard-coded.
```

Comment:
Can you explain in a comment what issue prepulling images helps solve?

Reply:
added. PTAL